### ISSUE TITLE
BUG REPORT -- `service list` broken in 0.13.0

### ISSUE THREAD

**Issue by lachie83** [kubernetes/minikube#881]
This is a BUG report

*This works in minikube 0.12.2
```shell
$ minikube version
minikube version: v0.12.2
$ minikube service list --namespace kube-system
|-------------|----------------------|-----------------------------|
|  NAMSEPACE  |         NAME         |             URL             |
|-------------|----------------------|-----------------------------|
| kube-system | kube-dns             | No node port                |
| kube-system | kubernetes-dashboard | http://192.168.99.100:30000 |
|-------------|----------------------|-----------------------------|
```
* The same command on 0.13.0 is now failing
```shell
$ minikube version
minikube version: v0.13.0
$ minikube service list --namespace kube-system
Error, attempted to generate service url with nil --format template
Check that minikube is running and that you have specified the correct namespace (-n flag) if required.
```
**Environment**:
- **OS** OSX 10.11.6
- **VM Driver** virtualbox
- **Docker version** 1.12.3
- **Install tools**:
- **Others**:

**Comment by aaron-prindle** [comment]
This issue is caused from a format string not getting set to its default value.  This PR should resolve the issue: #882

**Comment by lachie83** [comment]
Thank @aaron-prindle. Is there any way to work around this in the interim?

**Comment by lachie83** [comment]
w/o building from source.

**Comment by aaron-prindle** [comment]
Yes, this should work:
minikube service list --format="http://{{.IP}}:{{.Port}}" --namespace kube-system

**Comment by lachie83** [comment]
Yeah that's what I thought. No dice.

```shell
$ ./minikube_v0.13.0 service list --format="http://{{.IP}}:{{.Port}}" --namespace kube-system
Error, attempted to generate service url with nil --format template
Check that minikube is running and that you have specified the correct namespace (-n flag) if required.
```

**Comment by aaron-prindle** [comment]
Sorry you're correct, I tried that on the build w/ the PR added.  Service list does not have a --format flag in this v0.13.0 release so I do not think there will be a way to change the nil value.

**Comment by lachie83** [comment]
No worries. Thanks for your help @aaron-prindle

**Comment by dlorenc** [comment]
This should be fixed, but we'll need to release a 0.13.1 with the fix. We'll try to get that out Monday.
